.devops/tools.sh

```diff
@@ -7,15 +7,12 @@ arg1="$1"
 # Shift the arguments to remove the first one
 shift
 
-# Join the remaining arguments into a single string
-arg2="$@"
-
 if [[ "$arg1" == '--convert' || "$arg1" == '-c' ]]; then
-    python3 ./convert.py "$arg2"
+    exec python3 ./convert.py "$@"
```

Comment:
why `exec`, isnt just `$@` (without the quotes) enough?

Reply:
It's not really related to the issue, I can remove them.

I just thought it would be better.

Reply:
I just wasnt sure what they where used for :smile: 
do we need the quotes?

Reply:
exec causes the script's process to be replaced by the new one instead of forking, which avoids having an unnecessary bash process idling. We need the quotes to prevent splitting the arguments on spaces.

Reply:
It uses the Unix exec syscall to not just run the command and then return to the script but to replace the calling executable with the command that is run. That means Bash will not stay loaded in memory while main is running.

Quotes are needed when passing arguments with spaces (like `-p "Hello, world"`).